app/controllers/authenticate_controller.rb

```diff
@@ -31,6 +31,16 @@ def status
     render(status_failure_response(e))
   end
 
+  def authn_jwt_status
+    Authentication::AuthnJwt::ValidateStatus.new.call(
+      authenticator_status_input: status_input("authn-jwt")
+    )
```

Comment:
in `authenticate_jwt` we add `params[:authenticator] = "authn-jwt"` before calling `status_input` and this is how we overcome the fact that it is not preset. can we do the same here instead of sending a hard-coded `authn-jwt` to the method?